packages/lib/hooks/useSlots.ts

```diff
@@ -0,0 +1,8 @@
+type UseSlotsProps = {
+  boundaries: {
+    startTime: Date;
+    endTime: Date;
+  }[];
+};
+
+export const useSlots = ({ boundaries }: UseSlotsProps) => {};
```

Comment:
Is this empty on purpose @emrysal ?

Reply:
I think this is like, very early designing when I was still thinking "boundaries" are the way to go. will delete (it's unused)